src/sasquatchbackpack/cli.py

```diff
@@ -67,7 +70,7 @@ def check_coords(
             f"""Your provided longitude ({value[1]}) is too low.
  The minimum is -180."""
         )
-    elif value[1] > 180:
+    if value[1] > 180:
         raise click.BadParameter(
             f"""Your provided longitude ({value[1]}) is too high.
  The maximum is 180."""
```

Comment:
You can simplify the above using single if statements and making the code more readable by introducing the `latitude` and `longitude` variables:
```python
    latitude, longitude = value
    if latitude < -90.0 or latitude > 90.0:
        raise click.BadParameter(
            f"Your provided latitude ({latitude}) is out of bounds."
            "The range is -90 to 90."
        )
        
    if longitude < -180.0 or longitude > 180.0:
        raise click.BadParameter(
            f"Your provided longitude ({longitude}) is out of bounds."
            "The range is -180 to 180."
        )
```